tools/run-task.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+# This temporary script runs an arbitrary task using an existing task defintion
+# in a new container, using ECS RunTask. This script will be replaced by
+# features built into the GDS CLI.
+
+set -eu
+
+OPTIND=1 # Reset in case getopts has been used previously in the shell.
+
+cluster="task_runner"
+govuk_env="test"
+
+function show_help {
+  echo "Usage: cd govuk-infrastructure && gds aws govuk-tools-internal-admin ./tools/run-rake-task -a frontend rake db:migrate"
+}
+
+while getopts :a:c:e:h opt; do
+    case $opt in
+      h) show_help && exit 0;;
+      a)
+        application=${OPTARG} >&2
+        ;;
+      e)
+        govuk_env=${OPTARG:-$govuk_env} >&2
+        ;;
+      :)
+        echo "Option -$OPTARG requires an app as the argument. e.g. -a frontend" >&2
+        exit 1
+        ;;
+    esac
+done
+
+shift $((OPTIND-1))
+[ "${1:-}" = "--" ] && shift
+command=$@
+
+if [ -z "${command}" ]; then echo "[Error] You must set the application arg" && show_help && exit 1; fi
+if [ -z "${command}" ]; then echo "[Error] You must set provide a command" && show_help && exit 1; fi
+
+echo "[Warning] This is a temporary tool for use by the replatforming team."
+
+root_dir=$(pwd)
```

Comment:
As a rule of thumb it's a good idea to use quotes in case the variable contains whitespace. (and might as well use $PWD rather than a subshell)

```suggestion
root_dir="${PWD}"
```

Reply:
Thanks for the tip!